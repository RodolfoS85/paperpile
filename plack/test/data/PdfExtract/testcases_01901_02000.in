---
file: ../../../PDFREPOSITORY/001901.pdf
---
file: ../../../PDFREPOSITORY/001902.pdf
---
file: ../../../PDFREPOSITORY/001903.pdf
---
file: ../../../PDFREPOSITORY/001904.pdf
---
file: ../../../PDFREPOSITORY/001905.pdf
---
file: ../../../PDFREPOSITORY/001906.pdf
---
file: ../../../PDFREPOSITORY/001907.pdf
---
file: ../../../PDFREPOSITORY/001908.pdf
---
file: ../../../PDFREPOSITORY/001909.pdf
---
file: ../../../PDFREPOSITORY/001910.pdf
---
file: ../../../PDFREPOSITORY/001911.pdf
---
file: ../../../PDFREPOSITORY/001912.pdf
---
file: ../../../PDFREPOSITORY/001913.pdf
---
file: ../../../PDFREPOSITORY/001914.pdf
---
file: ../../../PDFREPOSITORY/001915.pdf
---
file: ../../../PDFREPOSITORY/001916.pdf
---
file: ../../../PDFREPOSITORY/001917.pdf
---
file: ../../../PDFREPOSITORY/001918.pdf
---
file: ../../../PDFREPOSITORY/001919.pdf
---
file: ../../../PDFREPOSITORY/001920.pdf
---
file: ../../../PDFREPOSITORY/001921.pdf
---
file: ../../../PDFREPOSITORY/001922.pdf
---
file: ../../../PDFREPOSITORY/001923.pdf
---
file: ../../../PDFREPOSITORY/001924.pdf
---
file: ../../../PDFREPOSITORY/001925.pdf
---
file: ../../../PDFREPOSITORY/001926.pdf
---
file: ../../../PDFREPOSITORY/001927.pdf
---
file: ../../../PDFREPOSITORY/001928.pdf
---
file: ../../../PDFREPOSITORY/001929.pdf
---
file: ../../../PDFREPOSITORY/001930.pdf
---
file: ../../../PDFREPOSITORY/001931.pdf
---
file: ../../../PDFREPOSITORY/001932.pdf
---
file: ../../../PDFREPOSITORY/001933.pdf
---
file: ../../../PDFREPOSITORY/001934.pdf
---
file: ../../../PDFREPOSITORY/001935.pdf
---
file: ../../../PDFREPOSITORY/001936.pdf
---
file: ../../../PDFREPOSITORY/001937.pdf
---
file: ../../../PDFREPOSITORY/001938.pdf
---
file: ../../../PDFREPOSITORY/001939.pdf
---
file: ../../../PDFREPOSITORY/001940.pdf
---
file: ../../../PDFREPOSITORY/001941.pdf
---
file: ../../../PDFREPOSITORY/001942.pdf
---
file: ../../../PDFREPOSITORY/001943.pdf
---
file: ../../../PDFREPOSITORY/001944.pdf
---
file: ../../../PDFREPOSITORY/001945.pdf
---
file: ../../../PDFREPOSITORY/001946.pdf
---
file: ../../../PDFREPOSITORY/001947.pdf
---
file: ../../../PDFREPOSITORY/001948.pdf
---
file: ../../../PDFREPOSITORY/001949.pdf
---
file: ../../../PDFREPOSITORY/001950.pdf
---
file: ../../../PDFREPOSITORY/001951.pdf
---
file: ../../../PDFREPOSITORY/001952.pdf
---
file: ../../../PDFREPOSITORY/001953.pdf
---
file: ../../../PDFREPOSITORY/001954.pdf
---
file: ../../../PDFREPOSITORY/001955.pdf
---
file: ../../../PDFREPOSITORY/001956.pdf
---
file: ../../../PDFREPOSITORY/001957.pdf
---
file: ../../../PDFREPOSITORY/001958.pdf
---
file: ../../../PDFREPOSITORY/001959.pdf
---
file: ../../../PDFREPOSITORY/001960.pdf
---
file: ../../../PDFREPOSITORY/001961.pdf
---
file: ../../../PDFREPOSITORY/001962.pdf
---
file: ../../../PDFREPOSITORY/001963.pdf
---
file: ../../../PDFREPOSITORY/001964.pdf
---
file: ../../../PDFREPOSITORY/001965.pdf
---
file: ../../../PDFREPOSITORY/001966.pdf
---
file: ../../../PDFREPOSITORY/001967.pdf
---
file: ../../../PDFREPOSITORY/001968.pdf
---
file: ../../../PDFREPOSITORY/001969.pdf
---
file: ../../../PDFREPOSITORY/001970.pdf
---
file: ../../../PDFREPOSITORY/001971.pdf
---
file: ../../../PDFREPOSITORY/001972.pdf
---
file: ../../../PDFREPOSITORY/001973.pdf
---
file: ../../../PDFREPOSITORY/001974.pdf
---
file: ../../../PDFREPOSITORY/001975.pdf
---
file: ../../../PDFREPOSITORY/001976.pdf
---
file: ../../../PDFREPOSITORY/001977.pdf
---
file: ../../../PDFREPOSITORY/001978.pdf
---
file: ../../../PDFREPOSITORY/001979.pdf
---
file: ../../../PDFREPOSITORY/001980.pdf
---
file: ../../../PDFREPOSITORY/001981.pdf
---
file: ../../../PDFREPOSITORY/001982.pdf
---
file: ../../../PDFREPOSITORY/001983.pdf
---
file: ../../../PDFREPOSITORY/001984.pdf
---
file: ../../../PDFREPOSITORY/001985.pdf
---
file: ../../../PDFREPOSITORY/001986.pdf
---
file: ../../../PDFREPOSITORY/001987.pdf
---
file: ../../../PDFREPOSITORY/001988.pdf
---
file: ../../../PDFREPOSITORY/001989.pdf
---
file: ../../../PDFREPOSITORY/001990.pdf
---
file: ../../../PDFREPOSITORY/001991.pdf
---
file: ../../../PDFREPOSITORY/001992.pdf
---
file: ../../../PDFREPOSITORY/001993.pdf
---
file: ../../../PDFREPOSITORY/001994.pdf
---
file: ../../../PDFREPOSITORY/001995.pdf
---
file: ../../../PDFREPOSITORY/001996.pdf
---
file: ../../../PDFREPOSITORY/001997.pdf
---
file: ../../../PDFREPOSITORY/001998.pdf
---
file: ../../../PDFREPOSITORY/001999.pdf
---
file: ../../../PDFREPOSITORY/002000.pdf
