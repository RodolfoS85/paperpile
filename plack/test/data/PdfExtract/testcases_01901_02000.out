---
authors: 'Lehner, Ben'
doi: 10.1242/jeb.002311
title: Modelling genotype-phenotype relationships and human disease with genetic interaction networks
---
authors: 'Sakai, M and Taniguchi, K and Mamoto, K and Ogawa, H and Tabata, M'
title: 'Immunostimulant effects of nucleotide isolated from yeast RNA on carp, Cyprinus carpio L'
---
authors: 'Levy-Coffman, Ellen'
title: 'We Are Not Our Ancestors: Evidence for Discontinuity between Prehistoric and Modern Europeans'
---
authors: 'Deboeck, Guido J.'
title: Genetic Diversity in Flemish Y-DNA
---
authors: 'Altman, Russ B.'
title: 'A Probabilistic Algorithm for Calculating Structure: Borrowing from Simulated Annealing'
---
authors: 'Maeda, Takuji and Kusakabe, Takahiro and Lee, Jae Man and Miyagawa, Yoshitaka and Koga, Katsumi and Kawaguchi, Yutaka'
title: Efficient Nonviral Gene Transfer Mediated by Polyethylenimine in an Insect Cell Line
---
authors: 'Itoh, Masanobu and Nishimura, Mika and Yukuhiro, Kenji and Yoshioka, Yasuhide and Miyata, Seiji and Yamaguchi, Masamitsu'
title: 'Duplicated alkaline phosphatase genes in the wild silkworm Bombyx mandarina inhabiting Japan: phylogenetic relationship to the orthologs in Bombyx mori'
---
authors: 'Cetin, Huseyin and Erler, Fedai and Yanikoglu, Atila'
title: 'Larvicidal activity of novaluron, a chitin synthesis inhibitor, against the housefly, Musca domestica'
---
authors: 'Namgail, T.'
title: 'WINTER HABITAT PARTITIONING BETWEEN ASIATIC IBEX AND BLUE SHEEP IN LADAKH, NORTHERN INDIA'
---
authors: 'Rolando, A. and Dondero, F. and Ciliento, E. and Laiolo, P.'
title: 'PASTORAL PRACTICES AND BIRD COMMUNITIES IN GRAN PARADISO NATIONAL PARK: MANAGEMENT IMPLICATIONS INTHE ALPS'
---
authors: 'Vigneault, Bernard'
doi: 10.1111/j.1529-8817.2005.04068.x
title: 'UPTAKE OF CADMIUM BY FRESHWATER GREEN ALGAE: EFFECTS OF pH AND AQUATIC HUMIC SUBSTANCES1'
---
authors: 'Khrustalev, Vladislav V. and Barkovsky, Eugene V.'
title: Main Pathways of Proteome Simplification in Alphaherpesviruses Under the Influence of the Strong Mutational GC-pressure
---
authors: 'Knight, Alan L. and Miliczky, Eugene'
title: 'Influence of Trap Colour on the Capture of Codling Moth (Lepidoptera: Tortricidae), Honeybees, and Non-target Flies'
---
authors: 'VAN HERK, W. G. and Raworth, D. A. and Gillies, S. L.'
title: 'Movement of Ericaphis fimbriata (Hemiptera: Aphididae) apterae on blueberry'
---
authors: 'Cairns, David M. and Lafon, Charles W. and Waldron, John D. and Tchakerian, Maria and Coulson, Robert N. and Klepzig, Kier D. and Birt, Andrew G. and Xi, Weimin'
doi: 10.1007/s10980-008-9198-7
title: Simulating the reciprocal interaction of forest landscape structure and southern pine beetle herbivory using LANDIS
---
authors: 'Agardy, Tundi'
title: Introduction to Marine Conservation Biology
---
authors: 'Beghalia, M and Ghalem, S and Allali, H and Belouatek, A and Marouf, A'
title: Screening for Anti-crystallisation Calcium Oxalate Urolithiasis Activity in Algerian Plants
---
authors: 'Abdullah, Janna Ong and Khairoji, Khairul Anuar and Mahmood, Maziah'
title: Anthocyanin Stability Studies in Tibouchina semidecandra
---
authors: 'Dash, Nihar and AL-Zarouni, Mansour and Al-Kous, Nora and Al-Shehhi, Fatma and Al-Najjar, Jalila and Senok, Abiola and Panigrahi, Debadatta'
title: 'Distribution and Resistance Trends of Community Associated Urinary Tract Pathogens in Sharjah, UAE'
---
authors: 'Corbit, Kevin C. and Soh, JAE-WON and Yoshida, Keiko and Eves, Eva M. and BERNARD WEINSTEIN, I. and Rosner, Marsha Rich'
title: Different Protein Kinase C Isoforms Determine Growth Factor Specificity in Neuronal Cells
---
authors: 'Wang, Chun-Chao and Cirit, Murat and Haugh, Jason M'
doi: 10.1038/msb.2009.4
title: PI3K-dependent cross-talk interactions converge with Ras as quantifiable inputs integrated by Erk
---
authors: 'Fayed, N. and Modrego, P. J. and Eiras, J.'
title: The Relationship Between Location and Prognosis in Brain-Stem Tumors
---
authors: 'Cardoso, Danon Clemes and Cristiano, Maykon Passos and Arent, Camila Orlandi and Sciences, Biological'
title: 'Development of New Didactic Materials for Teaching Science and Biology: The Importance of the New Education Practices'
---
authors: 'Gatherer, Derek'
title: 'TreeAdder: A Tool to Assist the Optimal Positioning of a New Leaf into an Existing Phylogenetic Tree'
---
authors: 'Ramanathan, K. and Shanthi, V. and Sethumadhavan, Rao'
title: Significance of CH/ Interactions on the Stability of Therapeutic Proteins
---
authors: 'Fu, Yong-Bi and Qiu, Jie and Wilmshurst, John F.'
title: 'Interpolating Genetic Variation in Natural Populations: A Case Study of Plains Rough Fescue (Festuca hallii)'
---
authors: 'Eriksen, Elena and Prozorkevich, Dmitry and Dingsør, Gjert E.'
title: An Evaluation of 0-Group Abundance Indices of Barents Sea Fish Stocks
---
authors: 'Fluhr, Herbert and Krenzer, Stefanie and Zygmunt, Marek'
doi: 10.1111/j.1447-0578.2008.00213.x
title: 'Different regulation of tissue inhibitors of metalloproteinases-1, -2 and -3 in human endometrial stromal cells during decidualization in vitro'
---
authors: 'Fujinoki, Masakatsu and Imaizumi, Yoshie and Ohtake, Hideki and Yamaoka, Sadao'
title: Effects of dibromoacetic acid on murine spermatozoa and testis
---
authors: 'Adeokun, O. A. and Adereti, F. O. and Akanni, K. A.'
title: "Children's Involvement in Fish Production: A Strategy for Poverty Alleviation in Waterside Local Government Area, Ogun State, Nigeria"
---
authors: 'Lashari, K. H. and Sahato, G. A. and Korai, A. L. and Kazi, T. G.'
title: 'Taxonomic Study of Chroocophyceae (Cyanophyta) in Keenjhar Lake, Distt: Thatta, Sindh, Pakistan'
---
authors: 'Ali, Aisha H. and Taalab, A. S.'
title: 'Effect of Natural And/or Chemical Potassium Fertilizers on Growth, Bulbs Yield and Some Physical and Chemical Constituents of Onion (Allium cepa, L.)'
---
authors: 'Prochnow, Shane J. and White, Joseph D. and Scott, Thad and Filstrup, Christopher'
doi: 10.1155/2007/12571
title: Small Reservoir Impact on Simulated Watershed-Scale Nutrient Yield
---
authors: 'Pennacchio, Marcello and Jefferson, Lara V. and Havens, Kayri'
doi: 10.1155/2007/65083
title: Allelopathic Effects of Plant-Derived Aerosol Smoke on Seed Germination of Arabidopsis thaliana (L.) Heynh
---
authors: 'Sussman, Benjamin J. and Townsend, Dave and Ivanov, Misha Yu. and Stolow, Albert'
doi: 10.1126/science.1132289
title: Dynamic Stark Control of Photochemical Processes
---
authors: 'Mindlin, S. Z. and Soina, V. S. and Petrova, M. A. and Gorlenko, Zh. M.'
doi: 10.1134/S1022795408010043
title: Isolation of Antibiotic Resistance Bacterial Strains from Eastern Siberia Permafrost Sediments
---
authors: 'blom, Tobias and Jones, Sia and Wood, Laura D. and Williams Parsons, D. and Lin, Jimmy and Barber, Thomas D. and Mandelker, Diana and Leary, Rebecca J. and Ptak, Janine and Silliman, Natalie and Szabo, Steve and Buckhaults, Phillip and Farrell, Christopher and Meeh, Paul and Markowitz, Sanford D. and Willis, Joseph and Dawson, Dawn and Willson, James K. V. and Gazdar, Adi F. and Hartigan, James and Wu, Leo and Liu, Changsheng and Parmigiani, Giovanni and Park, Ben Ho and Bachman, Kurtis E. and Papadopoulos, Nickolas and Vogelstein, Bert and Kinzler, Kenneth W. and Velculescu, Victor E.'
doi: 10.1126/science.1133427
title: The Consensus Coding Sequences of Human Breast and Colorectal Cancers
---
authors: 'Neubert, Michael G. and Herrera, Guillermo E.'
doi: 10.1007/s12080-007-0009-6
title: Triple benefits from spatial resource management
---
authors: 'Katagiri, Fumiaki and Sato, Masanao'
doi: 10.1007/s11248-007-9105-z
title: Expression profiles as detailed snapshots of biological states
---
authors: 'Zaefyzadeh, Mohammad and Quliyev, Rauf Alakbar and Babayeva, Sevda Mashalla and Abbasov, Mehraj Ali'
title: The Effect of the Interaction between Genotypes and Drought Stress on the Superoxide Dismutase and Chlorophyll Content in Durum Wheat Landraces
---
authors: 'Saadoun, Ismail and Wahiby, Layali and Ababneh, Qotaiba and Jaradat, Ziad and Massadeh, Muhannad and Al-Momani, Fouad'
doi: 10.1007/s11274-007-9451-2
title: Recovery of soil streptomycetes from arid habitats in Jordan and their potential to inhibit multi-drug resistant Pseudomonas aeruginosa pathogens
---
arxivid: math/0602612v2
authors: 'Flandoli, Franco and Romito, Marco'
title: MARKOV SELECTIONS FOR THE 3D STOCHASTIC NAVIER-STOKES EQUATIONS
---
authors: 'Laferriere, Alain and Gautheret, Daniel and Cedergren, Robert'
title: An RNA pattern matching program with enhanced performance and portability
---
authors: 'Alfonsi, Giancarlo and Passoni, Giuseppe and Pancaldo, Lea and Zampaglione, Domenico'
title: 'A SPECTRAL-FINITE DIFFERENCE SOLUTION OF THE NAVIER- STOKES EQUATIONS IN THREE DIMENSIONS'
---
authors: 'Gautheret, Daniel and Major, Francois and Cedergren, Robert'
title: 'Pattern searching/alignment with RNA primary and secondary structures: an effective descriptor for tRNA'
---
arxivid: hep-th/9411028v1
authors: 'Polchinski, Joseph'
title: WHAT IS STRING THEORY?
---
authors: 'Conte, D. and Foggia, P. and Sansone, C. and Vento, M.'
title: THIRTY YEARS OF GRAPH MATCHING IN PATTERN RECOGNITION
---
authors: 'Barnard, John M.'
title: 'Substructure Searching Methods: Old and Newt'
---
authors: 'Amaral, Paulo P. and Dinger, Marcel E. and Mercer, Tim R. and Mattick, John S.'
doi: 10.1126/science.1155472
title: The Eukaryotic Genome as an RNA Machine
---
authors: 'Amores, Angel and Force, Allan and Yan, Yi-Lin and Joly, Lucille and Amemiya, Chris and Fritz, Andreas and Ho, Robert K. and Langeland, James and Prince, Victoria and Wang, Yan-Ling and Westerfield, Monte and Ekker, Marc and Postlethwait, John H.'
title: Zebrafish hox Clusters and Vertebrate Genome Evolution
---
authors: 'Andersen, Ebbe S. and LIND-THOMSEN, Allan and Knudsen, Bjarne and Kristensen, Susie E. and Havgaard, Jakob H. and Torarinsson, Elfar and Larsen, Niels and Zwieb, Christian and Sestoft, Peter and Kjems, JØRGEN and Gorodkin, Jan'
doi: 10.1261/rna.215407
title: Semiautomated improvement of RNA alignments
---
authors: 'Anwar, Mohammad and Nguyen, Truong and Turcotte, Marcel'
doi: 10.1186/1471-2105-7-244
title: Identification of consensus RNA secondary structures using suffix arrays
---
authors: 'Bairoch, Amos and Apweiler, Rolf'
title: The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000
---
authors: 'Beldade, Patrícia and Brakefield, Paul M.'
doi: 10.1038/nrg818
title: THE GENETICS AND EVO-DEVO OF BUTTERFLY WING PATTERNS
---
authors: 'Berg, Johannes and Willmann, Stana and Lssig, Michael'
doi: 10.1186/1471-2148-4-42
title: Adaptive evolution of transcription factor binding sites
---
authors: 'Bortoluzzi, Stefania and Coppe, Alessandro and Bisognin, Andrea and Pizzi, Cinzia and Danieli, Gian Antonio'
doi: 10.1186/1471-2105-6-121
title: A multistep bioinformatic approach detects putative regulatory elements in gene promoters
---
authors: 'Brennecke, Julius and Stark, Alexander and Russell, Robert B. and Cohen, Stephen M.'
doi: 10.1371/journal.pbio.0030085
title: Principles of MicroRNA-Target Recognition
---
authors: 'Brown, Daniel G. and Li, Ming and Ma, Bin'
title: A TUTORIAL OF RECENT DEVELOPMENTS IN THE SEEDING OF LOCAL ALIGNMENT
---
authors: 'Carvalho, Luis E. and Lawrence, Charles E.'
doi: 10.1073/pnas.0712329105
title: Centroid estimation in discrete high-dimensional spaces with applications in biology
---
authors: 'Chan, Eugene Y.'
doi: 10.1007/978-1-60327-411-1_5
title: 'Next-Generation Sequencing Methods: Impact of Sequencing Accuracy on SNP Discovery'
---
authors: 'Chevalier, Clément and Geissmann, Thomas and Helfer, Anne-Catherine and Romby, Pascale'
doi: 10.1007/978-1-59745-558-9_16
title: Probing mRNA Structure and sRNA-mRNA Interactions in Bacteria Using Enzymes and Lead(II)
---
authors: 'Clark, Andrew G. and Glanowski, Stephen and Nielsen, Rasmus and Thomas, Paul D. and Kejariwal, Anish and Todd, Melissa A. and Tanenbaum, David M. and Civello, Daniel and Lu, Fu and Murphy, Brian and Ferriera, Steve and Wang, Gary and Zheng, Xianqgun and White, Thomas J. and Sninsky, John J. and Adams, Mark D. and Cargill, Michele'
doi: 10.1126/science.1088821
title: Inferring Nonneutral Evolution from Human-Chimp-Mouse Orthologous Gene Trios
---
authors: 'Cordaux, Richard and Udit, Swalpa and Batzer, Mark A. and Feschotte, Cedric'
doi: 10.1073/pnas.0601161103
title: Birth of a chimeric primate gene by capture of the transposase gene from a mobile element
---
authors: 'Crothers, Donald M.'
title: 'DNA curvature and deformation in protein-DNA complexes: A step in the right direction'
---
authors: 'Davletov, Bazbek and Jiménez, José L'
title: Sculpting a domain by splicing
---
authors: 'de la Mata, Manuel and Kornblihtt, Alberto R'
doi: 10.1038/nsmb1155
title: RNA polymerase II C-terminal domain mediates regulation of alternative splicing by SRp20
---
authors: 'Dermitzakis, Emmanouil T. and Reymond, Alexandre and Lyle, Robert and Scamuffa, Nathalie and Ucla, Catherine and Deutsch, Samuel and Stevenson, Brian J. and Flegel, Volker and Bucher, Philipp and Jongeneel, C. Victor and Antonarakis, Stylianos E.'
title: Numerous potentially functional but non-genic conserved sequences on human chromosome 21
---
authors: 'Ding, YE'
doi: 10.1261/rna.2274106
title: Statistical and Bayesian approaches to RNA secondary structure prediction
---
authors: 'Dou, Yimeng and FOX-WALSH, Kristi L. and Baldi, Pierre F. and Hertel, Klemens J.'
doi: 10.1261/rna.151106
title: Genomic splice-site analysis reveals frequent alternative splicing close to the dominant splice site
---
authors: 'Drews, Jurgen'
doi: 10.1126/science.287.5460.1960
title: 'Drug Discovery: A Historical Perspective'
---
authors: 'Dubey, Ashok K. and Baker, Carol S. and Romeo, Tony and Babitzke, Paul'
doi: 10.1261/rna.2990205
title: RNA sequence and secondary structure participate in high-affinity CsrA-RNA interaction
---
authors: 'Fairbrother, William G. and Holste, Dirk and Burge, Christopher B. and Sharp, Phillip A.'
doi: 10.1371/journal.pbio.0020268
title: Single Nucleotide Polymorphism-Based Validation of Exonic Splicing Enhancers
---
authors: 'German, Marcelo A and Pillay, Manoj and Jeong, Dong-Hoon and Hetawal, Amit and Luo, Shujun and Janardhanan, Prakash and Kannan, Vimal and Rymarquis, Linda A and Nobuta, Kan and German, Rana and De Paoli, Emanuele and Lu, Cheng and Schroth, Gary and Meyers, Blake C and Green, Pamela J'
doi: 10.1038/nbt1417
title: Global identification of microRNA-target RNA pairs by parallel analysis of RNA ends
---
authors: 'Gesell, Tanja and Washietl, Stefan'
doi: 10.1186/1471-2105-9-248
title: Dinucleotide controlled null models for comparative RNA gene prediction
---
authors: 'Baquero, F. and Berche, P. and Bloecker, H. and Brandt, P. and Chakraborty, T. and Charbit, A. and Chetouani, F. and Couve, E. and Kaerst, U. and Kreft, J. and Kuhn, M. and Kunst, F. and Kurapkat, G. and Madueno, E. and Maitournam, A. and Mata Vicente, J. and Ng, E. and Nedjari, H. and Nordsiek, G. and Novella, S. and de Pablos, B. and -C. Perez-Diaz, J. and Purcell, R. and Remmel, B. and Rose, M. and Schlueter, T. and Simoes, N. and Tierrez, A. and -A. Vazquez-Boland, J. and Voss, H. and Wehland, J. and Cossart, P.'
doi: 10.1126/science.1063447
title: Comparative Genomics of Listeria Species
---
authors: 'Gorodkin, Jan and Stricklin, Shawn L. and Stormo, Gary D.'
title: Discovering common stem-loop motifs in unaligned RNA sequences
---
authors: 'Gramada, Apostol and Bourne, Philip E.'
doi: 10.1186/1471-2105-7-242
title: Multipolar representation of protein structure
---
authors: 'Gruber, Andreas R. and FINDEIß, Sven and Washietl, Stefan and Hofacker, Ivo L. and Stadler, Peter F.'
title: 'RNAZ 2.0: IMPROVED NONCODING RNA DETECTION'
---
authors: 'Hall, Kathleen B'
title: RNA-protein interactions
---
authors: 'Hammell, Molly and Long, Dang and Zhang, Liang and Lee, Andrew and Carmack, C Steven and Han, Min and Ding, Ye and Ambros, Victor'
doi: 10.1038/NMETH.1247
title: 'mirWIP: microRNA target prediction based on microRNA-containing ribonucleoprotein-enriched transcripts'
---
authors: 'Hardison, Ross C. and Oeltjen, John and Miller, Webb'
title: 'Long Human-Mouse Sequence Alignments Reveal Novel Regulatory Elements: A Reason to Sequence the Mouse Genome'
---
authors: 'Hatton, Allyson R. and Subramaniam, Vaidyanathan and Lopez, A. Javier'
title: Generation of Alternative Ultrabithorax Isoforms and Stepwise Removal of a Large Intron by Resplicing at Exon-Exon Junctions
---
authors: 'Heilig, Roland and Eckenberg, Ralph and Petit, Jean-Louis and Fonknechten, Nuria and Da Silva, Corinne and Cattolico, Laurence and Levy, Michael and Barbe, Valerie and de Berardinis, Veronique and Ureta-Vidal, Abel and Pelletier, Eric and Vico, Virginie and Anthouard, Veronique and Rowen, Lee and Madan, Anup and Qin, Shizhen and Sun, Hui and Du, Hui and Pepin, Kymberlie and Artiguenave, Francois and Robert, Catherine and Cruaud, Corinne and Bruls, Thomas and Jaillon, Olivier and Friedlander, Lucie and Samson, Gaelle and Brottier, Philippe and Cure, Susan and Segurens, Beatrice and Aniere, Franck and Samain, Sylvie and Crespeau, Herve and Abbasi, Nissa and Aiach, Nathalie and Boscus, Didier and Dickhoff, Rachel and Dors, Monica and Dubois, Ivan and Friedman, Cynthia and Gouyvenoux, Michel and James, Rose and Madan, Anuradha and Mairey-Estrada, Barbara and Mangenot, Sophie and Martins, Nathalie and Menard, Manuela and Oztas, Sophie and Ratcliffe, Amber and Shaffer, Tristan and Trask, Barbara and Vacherie, Benoit and Bellemere, Chadia and Belser, Caroline and Besnard-Gonnet, Marielle and Bartol-Mavel, Delphine and Boutard, Magali and Briez-Silla, Stephanie and Combette, Stephane and Dufosse-Laurent, Virginie and Ferron, Carolyne and Lechaplais, Christophe and Louesse, Claudine and Muselet, Delphine and Magdelenat, Ghislaine and Pateau, Emilie and Petit, Emmanuelle and Sirvain-Trukniewicz, Peggy and Trybou, Arnaud and Vega-Czarny, Nathalie and Bataille, Elodie and Bluet, Elodie and Bordelais, Isabelle and Dubois, Maria and Dumont, Corinne and Guerin, Thomas and Haffray, Sebastien and Hammadi, Rachid and Muanga, Jacqueline and Pellouin, Virginie and Robert, Dominique and Wunderle, Edith and Gauguet, Gilbert and Roy, Alice and Sainte-Marthe, Laurent and Verdier, Jean and Verdier-Discala, Claude and Hillier, LaDeana and Fulton, Lucinda and McPherson, John and Matsudak, Fumihiko and Wilson, Richard and Scarpelli, Claude and Gyapay, Gabor and Wincker, Patrick and Saurin, William and Quetier, Francis and Waterston, Robert and Hood, Leroy and Weissenbach, Jean'
title: The DNA sequence and analysis of human chromosome 14
---
authors: 'Henkin, Tina M.'
doi: 10.1007/978-1-59745-558-9_15
title: RNA-Dependent RNA Switches in Bacteria
---
authors: 'Holm, Liisa and Sander, Chris'
title: 'The FSSP database: fold classification based on structure-structure alignment of proteins'
---
authors: 'Honig, Arnd and Auboeuf, Didier and Parker, Marjorie M. and OMalley, Bert W. and Berget, Susan M.'
doi: 10.1128/MCB.22.16.5698-5707.2002
title: Regulation of Alternative Splicing by the ATP-Dependent DEAD-Box RNA Helicase p72
---
authors: 'Hüttenhofer, Alexander and Schattner, Peter'
title: 'The principles of guiding by RNA: chimeric RNA-protein enzymes'
---
authors: 'Hwang, Dick G. and Green, Phil'
doi: 10.1073/pnas.0404142101
title: Bayesian Markov chain Monte Carlo sequence analysis reveals varying neutral substitution patterns in mammalian evolution
---
authors: 'Jiang, Tianzi'
doi: 10.1063/1.1592796
title: Protein folding simulations of the hydrophobic-hydrophilic model by combining tabu search with genetic algorithms
---
authors: 'Kent, W. James and Sugnet, Charles W. and Furey, Terrence S. and Roskin, Krishna M. and Pringle, Tom H. and Zahler, Alan M. and Haussler, David'
doi: 10.1101/gr.229102
title: The Human Genome Browser at UCSC
---
authors: 'Keys, David N. and Lewis, David L. and Selegue, Jane E. and Pearson, Bret J. and Goodrich, Lisa V. and Johnson, Ronald L. and Gates, Julie and Scott, Matthew P. and Carroll, Sean B.'
title: Recruitment of a hedgehog Regulatory Circuit in Butterfly Eyespot Evolution
---
authors: 'Khanam, Tasneem and Muddashetty, Ravi Sondekoppa and Kahvejian, Avak and Sonenberg, Nahum and Brosius, Juergen'
title: Poly(A)-Binding Protein Binds to A-Rich Sequences via RNA-Binding Domains 1+2 and 3+4
---
authors: 'Kirkpatrick, S. and Gelatt, Jr., C. D. and Vecchi, M. P.'
title: Optimization by Simulated Annealing
---
authors: 'Krol, Alain'
title: Evolutionarily different RNA motifs and RNA-protein complexes to achieve selenoprotein synthesis
---
authors: 'Kryukov, Gregory V. and Castellano, Sergi and Novoselov, Sergey V. and Lobanov, Alexey V. and Zehtab, Omid and Guigo, Roderic and Gladyshev, Vadim N.'
title: Characterization of Mammalian Selenoproteomes
---
authors: 'Langenberger, D. and BERMUDEZ-SANTANA, C. I. and Stadler, P. F. and Hoffmann, S.'
title: IDENTIFICATION AND CLASSIFICATION OF SMALL RNAS IN TRANSCRIPTOME SEQUENCE DATA
---
authors: 'Larkin, M. A. and Blackshields, G. and Brown, N. P. and Chenna, R. and McGettigan, P. A. and McWilliam, H. and Valentin, F. and Wallace, I. M. and Wilm, A. and Lopez, R. and Thompson, J. D. and Gibson, T. J. and Higgins, D. G.'
doi: 10.1093/bioinformatics/btm404
title: Clustal W and Clustal X version 2.0
---
authors: 'Li, Ying and Bor, Yeou-cherng and Misawa, Yukiko and Xue, Yuming and Rekosh, David and Hammarskjold, Marie-Louise'
doi: 10.1038/nature05107
title: An intron with a constitutive transport element is retained in a Tap messenger RNA
---
authors: 'Liang, Han and Li, WEN-HSIUNG'
doi: 10.1261/rna.634607
title: MicroRNA regulation of human protein-protein interaction network
---
authors: 'Lim, Lee P. and Glasner, Margaret E. and Yekta, Soraya and Burge, Christopher B. and Bartel, David P.'
title: Vertebrate MicroRNA Genes
